Subjet_BehavData_v5.py

```diff
@@ -1 +1,312 @@
+import numpy as np
+import pandas as pd
+import os, re
+import glob
```

Comment:
Now, you need to use glob to get all the files.
Something like this:
allMainFiles = glob.glob('Output-Responses-Encoding_CIMAQ*txt')
id = allMainFiles[0].split('_')[3]